### ISSUE TITLE
Issue on hovering

### ISSUE THREAD

**Issue by pepeu93** [zero-plusplus/vscode-autohotkey-debug#196]
Hello, looks like the `ECONNRESET` issue was probably gone, I didn't get it anymore in this last version.

However, now I'm getting a weird error when hovering objects:
When I hover `main` that is a subobject of `Ct_Object`, this is the result:

![image](https://user-images.githubusercontent.com/82878129/151460906-bd9d79d6-6fed-40e9-aad9-edc9e66e6407.png)

There's no `width` variable in this object.

And when I try to see the value of `main` from the `Ct_Object`.

It displays a loading icon that continues forever.

After some time the debugger exited with:
```
TypeError: u.property?.find is not a function
Debugging disconnected. AutoHotkey script is continued.
```

**Comment by zero-plusplus** [comment]
Thanks for the report.

Does this issue occur in 1.10.2?
I released 1.10.2 yesterday, which fixes this issue.

If it still occurs, please provide us with the minimal code that reproduces this issue.

**Comment by pepeu93** [comment]
I update to 1.10.2 now it showing the values correctly, thank you!
